spark-plugins/docs/File-streamingsource.md

```diff
@@ -1,3 +1,18 @@
+<!--- 
```

Comment:
do these not show up in the markdown doc?

I guess we haven't been putting copyrights in these doc files, don't know that they're really needed here though.

Reply:
I've confirmed that they dont show up. 

I'm not sure if its needed but it was requested in one of the previous review comments